2024/json-ld-wg/index.html

```diff
@@ -144,97 +144,53 @@ <h1 id="title"><i class=todo>DRAFT</i> JSON-LD Working Group Charter</h1>
+          Use of JSON-LD has grown and expanded over the last decade (since it's first publication as a
+          <a href="https://www.w3.org/TR/json-ld-api1/">W3C Recommendation in 2014</a>).
+          JSON-LD remains a popular choice for ecoding <abbr title="Seach Engine Optimization">SEO</abbr>-focused
+          structured data in Web pages. JSON-LD has also become a foundational format for social media (ActivityStreams),
+          Internet of Things data (Web of Things Description), Verifiable Credentials, and Software Bill of Materials (SPDX).
+        </p>
+        <p>This growth in the applied use of JSON-LD has resulted in community interest in additional expressions of
+          JSON-LD's underlying Linked Data structure into other formats. YAML-LD has been created by the
+          <a href="https://www.w3.org/community/json-ld/">JSON for Linking Data Community Group</a> for easing human
+          authoring and providing a clearer path for using Linked Data applied to popular YAML-based documents such as
+          infrastructure as code, static site front matter, and API documentation formats. Additionally, a need has
+          arised in the Verifiable Credentials space for providing a more compressed expression of JSON-LD, and work on
+          CBOR-LD was begun in the JSON-LD CG.
+        </p>
+        <p>
+          The hope of this new more inclusively focused JSON-LD WG is to meet the demand of the broader
+          JSON-LD community by strengthening the foundational JSON-LD specifications and aligning it with the latest
+          RDF 1.2 specifications, growing the family of JSON-LD related formats which orbit around the JSON-LD
+          processing model, and specifing improved context retrieval methods for greater clarity and stability.
         </p>
       </div>
 
       <section id="scope" class="scope">
         <h2>Scope</h2>
 				<p>
 					The Working Group will extend the recommendations defining the JSON-LD specifications (i.e., <a href="https://www.w3.org/TR/json-ld11/">JSON-LD 1.1</a>, <a href="https://www.w3.org/TR/json-ld11-api/">JSON-LD 1.1 API</a>, <a href="https://www.w3.org/TR/json-ld11-framing/">JSON-LD 1.1 Framing</a>)
-          with the features introduced by <a href="https://www.w3.org/TR/rdf12-concepts">RDF-star</a>
+          with the features introduced by <a href="https://www.w3.org/TR/rdf12-concepts">RDF 1.2</a>
           in addition to open Errata and requested features
           as noted on the groups <a href="https://github.com/orgs/w3c/projects/84">Project Management Page</a>.
-          These specifications together provide a JSON format for Linked Open Data to interoperate at web-scale, in a method which is familiar to and usable by web-focused software engineers.
+          These specifications together provide a JSON format for Linked Data to interoperate in a method which is familiar to and usable by software engineers.
```

Comment:
```suggestion
          Together, these specifications provide a JSON format for Linked Data, enabling interoperation in a way which is familiar to and usable by software engineers.
```

Reply:
Resolved in https://github.com/json-ld/json-ld-wg-charter/pull/11/commits/257580c5ecb4e7e44a9dd203890ce19947fb491a